features/frontend/products.feature

```diff
@@ -13,7 +13,17 @@ Feature: Products
             | Clothing[TX1] > T-Shirts[TX2]     |
             | Clothing[TX1] > PHP T-Shirts[TX3] |
             | Clothing[TX1] > Gloves[TX4]       |
-          And there are following channels configured:
+          And there are following users:
+            | email       | password | enabled |
+            | bar@foo.com | foo1     | yes     |
+        And the following products exist:
+            | name             | price | taxons       | average rating |
+            | Super T-Shirt    | 19.99 | T-Shirts     | 0              |
+            | Black T-Shirt    | 18.99 | T-Shirts     | 0              |
+            | Sylius Tee       | 12.99 | PHP T-Shirts | 0              |
+            | Symfony T-Shirt  | 15.00 | PHP T-Shirts | 5              |
```

Comment:
Shouldn't average rating be calculated automatically? (I recall a fuzzy reasoning from original PR but have forgotten it)